MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
:smile:

On a serious note; I *do* think the personal email addresses make it harder to see which maintainers are affiliated with Docker, Inc. and which ones are not.

I remember @shykes mentioning this during a talk at DockerConEu that the open source project should be equally controlled by Docker, Inc. and external parties. Is this still a concern? I also realise this is not mentioned in the "rules" here, but possibly discussed by the DGAB?